{"name": "Проверка: нельзя создать двух одинаковых пользователей", "status": "passed", "description": "Проверяем, что пользователя с одинаковыми данными нельзя создать дважды", "start": 1721331648612, "stop": 1721331648783, "uuid": "840ebd12-030d-4c73-b686-4a6448d80258", "historyId": "e8d5a44b1cc50238bddd3305fdba3f31", "testCaseId": "e8d5a44b1cc50238bddd3305fdba3f31", "fullName": "tests.test_create_user.TestCreateUser#test_create_the_same_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "Mike"}, {"name": "thread", "value": "7692-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}